.github/renovate.json5

```
@@ -180,6 +180,16 @@
       depNameTemplate: "debian",
       datasourceTemplate: "docker",
       versioningTemplate: "debian",
+    },
+    {
+      fileMatch: [
+        "variables.sh$"
+      ],
+      matchStrings: [
+        "MYPY_VERSION =\\$\\{MYPY_VERSION:-(?<currentValue>.*?)\\}\\n"
```

sorry I added a space by mistake in my example. but the good news is that I tested it and it works well (after removing the space)
```suggestion
        "MYPY_VERSION=\\$\\{MYPY_VERSION:-(?<currentValue>.*?)\\}\\n"
```